README.md

```diff
@@ -21,9 +21,9 @@ let uri = "127.0.0.1:9042";
 
 let session: Session = SessionBuilder::new().known_node(uri).build().await?;
 
-let raw_iter = session.query_iter("SELECT a, b, c FROM ks.t", &[]).await?;
-let mut iter = raw_iter.into_typed::<(i32, i32, String)>();
-while let Some((a, b, c)) = iter.try_next().await? {
+let query_pager = session.query_iter("SELECT a, b, c FROM ks.t", &[]).await?;
+let mut stream = query_pager.rows_stream::<(i32, i32, String)>()?;
+while let Some((a, b, c)) = stream.try_next().await? {
     println!("a, b, c: {}, {}, {}", a, b, c);
 }
```

Comment:
Why not &str?

Reply:
`Stream` does not permit borrowed types.